views.py

```diff
@@ -347,3 +350,76 @@ def post(self, request):
                 'Unidentified Error',
                 status=status.HTTP_500_INTERNAL_SERVER_ERROR,
             )
+
+
+class WriteAppendMultipleObjects(APIView):
+    """
+    API endpoint that allows importing and exporting csv files
+    """
+
+    def get(self, request, *args, **kwargs):
+        """
+        Returns a blank csv file with xlsx extenstion for given model
+        :return: a blank csv file with xlsx extension for given model
+        """
+
+        model_name = request.GET.get('model')
+        if model_name is None:
+            return Response(
+                { 'Error': ['Model parameter is missing.'] },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+        try:
+            Model = swapper.load_model('faculty_biodata', model_name)
+
+            response = HttpResponse(content_type='text/csv')
+            response['Content-Disposition'] = f'attachment; filename={model_name}.xlsx'
+
+            all_fields = Model._meta.get_fields()
+            exclude_fields = ['id', 'datetime_created', 'datetime_modified', 'faculty_member', 'file']
+            column_headers = [field.name for field in all_fields if field.name not in exclude_fields]
+
+            writer = csv.writer(response)
+            writer.writerow(column_headers)
+
+            return response
+        except ImproperlyConfigured:
+            return Response(
+                { 'Invalid model' : ['Model doesn\'t exists.'] },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+    def post(self, request, *args, **kwargs):
+        data = request.data
+        model_name = data['model']
```

Comment:
I told you about the importance of https://github.com/IMGIITRoorkee/omniport-app-faculty-profile/pull/13/files#diff-92eb768276c8e0365258974b1341ecb5R366, right?